packages/core/core/src/Parcel.js

```diff
@@ -141,18 +141,22 @@ export default class Parcel {
     this.#watchEvents = new ValueEmitter();
     this.#disposable.add(() => this.#watchEvents.dispose());
 
-    this.#requestTracker = await RequestTracker.init({
-      farm: this.#farm,
-      options: resolvedOptions,
-    });
-
     this.#reporterRunner = new ReporterRunner({
       config: this.#config,
       options: resolvedOptions,
       workerFarm: this.#farm,
     });
     this.#disposable.add(this.#reporterRunner);
 
+    logger.verbose({
```

Comment:
Is this needed? I mean, it's verbose so it doesn't hurt, but just wondering whether it was a bit of debug code leftover.

Reply:
Sorry meant to leave a comment about this. I added this cause the reporters to initialize. That way we get accurate init timings from the reporter. We could remove this once we have a better method for tracking timing data in Parcel (e.g. performance marks, etc).